modules/oadp-creating-backup-hooks.adoc

```diff
@@ -27,32 +27,34 @@ spec:
       - name: <hook_name>
         includedNamespaces:
         - <namespace> <1>
-        excludedNamespaces:
+        excludedNamespaces: <2>
         - <namespace>
-        includedResources:
-        - pods <2>
-        excludedResources: []
-        labelSelector: <3>
+        includedResources: []
+        - pods <3>
+        excludedResources: [] <4>
+        labelSelector: <5>
           matchLabels:
             app: velero
             component: server
-        pre: <4>
+        pre: <6>
           - exec:
-              container: <container> <5>
+              container: <container> <7>
               command:
-              - /bin/uname <6>
+              - /bin/uname <8>
               - -a
-              onError: Fail <7>
-              timeout: 30s <8>
-        post: <9>
+              onError: Fail <9>
+              timeout: 30s <10>
+        post: <11>
 ...
 ----
-<1> Array of namespaces to which the hook applies. If this value is not specified, the hook applies to all namespaces.
-<2> Currently, pods are the only supported resource.
-<3> Optional: This hook only applies to objects matching the label selector.
-<4> Array of hooks to run before the backup.
-<5> Optional: If the container is not specified, the command runs in the first container in the pod.
-<6> Array of commands that the hook runs.
-<7> Allowed values for error handling are `Fail` and `Continue`. The default is `Fail`.
-<8> Optional: How long to wait for the commands to run. The default is `30s`.
-<9> This block defines an array of hooks to run after the backup, with the same parameters as the pre-backup hooks.
+<1> Optional: You can specify namespaces to which the hook applies. If this value is not specified, the hook applies to all namespaces.
```

Comment:
We generally don't write callout comments as instructions. I'd suggest "One or more namespaces...". I won't insist on this.